cli/package.json

```diff
@@ -24,6 +24,7 @@
   "license": "ISC",
   "type": "module",
   "dependencies": {
+    "@finos/calm-cli": "^0.2.3",
```

Comment:
Maybe misunderstanding - is this not a circular dependency?

Reply:
Yep, erroneous npm install when testing the module, removed